components/automate-ui/src/app/modules/user/user-table/user-table.component.html

```diff
@@ -1,11 +1,11 @@
-<app-authorized [allOf]="getPermissionsPath" [overrideVisible]="overridePermissionsCheck">
```

Comment:
Looks like this gets rid of the last consumer of `overrideVisible` so please remove that from authorized.component.ts.